tasks.py

```diff
@@ -267,29 +267,6 @@ def __init__(self, emotion, fn_tokenizer=bert_tokenizer):
         self.num_classes = 2
 
 
-    def get_iter(self, split, tokenizer, batch_size=16, shuffle=False, random_state=1, max_length=32):
```

Comment:
I don't think just deleting the single task get_iter solves our problem.
Now we would just ad 11 times the whole SemEval task if I am not mistaken.
The labels wil have the form [0,1,0,...,1,0] or similar but our code assumes one dimensional labels in many places.

Reply:
SemEval18SingleEmotionTask redefines the emotions array that it's used to create the labels:
https://github.com/baleato/atcs-project/blob/18baad02ddfe855302f77e9de4b7d0f6e20299ca/tasks.py#L263
https://github.com/baleato/atcs-project/blob/18baad02ddfe855302f77e9de4b7d0f6e20299ca/tasks.py#L226
So it does return only one label per example [0/1].